packages/nx-go/src/utils/normalize-options.helper.ts

```diff
@@ -24,7 +24,7 @@ export function normalizeOptions(
     projectDirectory,
     parsedTags,
     skipGoMod: !!options.skipGoMod,
-    npmScope: nxJson.npmScope,
```

Comment:
`npmScope` is no longer a configuration option in `nx.json` – this change is a best guess.